widgets/tree.py

```diff
@@ -55,20 +58,20 @@ def onDragOver(self, event):
 		try:
 			nodeType, srcKey = event.dataTransfer.getData("Text").split("/")
```

Comment:
@phorward We assign here two variables, but don't use them. Why?

Reply:
I don't know ;-)